doc/data.md

```diff
@@ -7,37 +7,43 @@ An observable key-value object used to cross bind to the url observable [can-rou
 key-value pairs, once [can-route.start] is called, changes in `route.data`'s
 properties will update the hash and vice-versa.
 
-```js
-import DefineMap from "can-define/map/map";
-import route from "can-route";
+  ```html
+  <mock-url></mock-url>
+  <script type="module">
+  import {DefineMap, route} from "can";
+  import "//unpkg.com/mock-url@^5.0.0/mock-url.mjs";
 
-route.data = new DefineMap( { page: "" } );
-route.register( "{page}" );
-route.start();
-```
+  route.data = new DefineMap( {page: ""} );
+  route.register( "{page}" );
+  route.start();
+
+  location.hash = "#!example";
+
+  setTimeout(()=> {
+    console.log( route.data ); //-> {page: "example"}
+  }, 100);
+  </script>
+  ```
+  @codepen
 
 
 @type {HTMLElement} If `route.data` is set to an element, its
 observable [can-view-model] will be used as the observable connected
-to the browser's hash.  
+to the browser's hash.
 
-```js
-import Component from "can-component";
-import route from "can-route";
-
-Component.extend( {
-	tag: "my-app",
-	autoMount: true,
-	ViewModel: { /* ... */ },
-	view: { /* ... */ }
-} );
-route.data = document.querySelector( "my-app" );
-route.register( "{page}" );
-route.start();
-```
+  <section class="warnings">
+    <div class="deprecated warning">
+    <h3>Deprecated</h3>
+    <div class="signature-wrapper">
+    <p>Assigning an [`HTMLElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) to `route.data` has been deprecated in favor of setting it to an observable. If you have any further questions please refer to the [guides/routing Routing] guide.
+    </div>
+    </div>
+  </section>
 
 @body
 
+For in-depth examples see the the [guides/routing Routing] guide.
+
 ## Background
 
 One of the biggest challenges in a complex application is getting all the different parts of the app to talk to each other simply, cleanly, and reliably.
```

Comment:
Wow, this video is so old.  Can you create an issue so that someone comes back to this?  We shoudn't be talking about an Application ViewModel anymore unless we are talking about `route.data` being "inside" the app VM

Reply:
Actually, I would just remove the entire use section.  Can you replace it with an example from the main `route-data.md` page where you set `route.data` to a `can-observe`?

In the signature, let people know that `route.data` defaults to a `DefineMap`.